src/test/java/com/datatorrent/netlet/DefaultEventLoopTest.java

```diff
@@ -300,4 +300,88 @@ public void registered(SelectionKey key)
     assertFalse(client2.isConnected());
   }
 
+  @Test
+  public void startStop() throws Exception
+  {
+    assertTrue(defaultEventLoop.isActive());
+    assertEquals(thread, defaultEventLoop.start());
+    defaultEventLoop.stop();
+    assertTrue(defaultEventLoop.isActive());
+    defaultEventLoop.stop();
+    thread.join();
+    assertFalse(defaultEventLoop.isActive());
+  }
+
+  @Test
+  public void startWithExtraStop() throws Exception
+  {
+    assertTrue(defaultEventLoop.isActive());
+    defaultEventLoop.stop();
+    thread.join();
```

Comment:
- No, I refer to using Timeout Rule.
- If you run locally you may interrupt test at any time.
- In some cases, it may be desirable to have an ability to see logs and analyze stack trace at the point when a test hangs.